python/lsst/pipe/tasks/mocks/simpleMapper.py

```diff
@@ -103,6 +103,34 @@ def makeButlerLocation(cls, path, dataId, mapper, suffix=None):
                                        dataId['imageOrigin'])
         return loc
 
+class ImageDPersistenceType(PersistenceType):
+    """Persistence type of ImageD images.
+    """
+
+    python = "lsst.afw.image.ImageD"
+    cpp = "ImageD"
+    storage = "FitsStorage"
+    ext = ".fits"
+    suffixes = ("_sub",)
+
+    @classmethod
+    def makeButlerLocation(cls, path, dataId, mapper, suffix=None):
+        """Method called by SimpleMapping to implement a map_ method; overridden to support subimages."""
```

Comment:
I do think the sub-image stuff is necessary. the co-addition code works with sub-images AFAIK.